pkg/porter/list.go

```diff
@@ -103,7 +103,7 @@ type DisplayInstallation struct {
 }
 
 type DisplayInstallationMetadata struct {
-	ResolvedParameters DisplayValues `json:"resolvedParameters", yaml:"resolvedParameters"`
+	ResolvedParameters DisplayValues `json:"resolvedParameters" yaml:"resolvedParameters"`
```

Comment:
@carolynvs 
This was broken earlier and hence the tag wasn't being used properly earlier. I think this has resulted in a broken test. I'm not sure what the best way to resolve this would be. 
We could regenerate the golden file, but I don't know how "breaking" this is.

Reply:
Thanks for fixing it! We can use `mage updateTestFiles` to regenerate the golden files for tests